common/compositor/nativesurface.cpp

```diff
@@ -79,7 +79,7 @@ bool NativeSurface::Init(ResourceManager *resource_manager, uint32_t format,
       HWCNativeHandle native_handle;
       resource_manager->GetNativeBufferHandler()->CreateBuffer(
           width_, height_, format, &native_handle, usage, 0);
-      if (!native_handle_) {
+      if (!native_handle) {
```

Comment:
This change is correct.  However, as a side comment, I'm wondering if the same call to CreateBuffer on line 54 needs to update a local 'HWCNativeHandle native_handle' similar to this snippet.  That way the 'native_handle_' class member could still maintain a valid handle throughout.

Reply:
I like that idea. I'll see if I can make it work.